src/backend/libpq/rangerrest.c

```diff
@@ -391,7 +363,7 @@ static size_t write_callback(char *contents, size_t size, size_t nitems,
         void *userp)
 {
     size_t realsize = size * nitems;
-    CURL_HANDLE curl = (struct curl_context *) userp;
+    CURL_HANDLE curl = (struct curl_context_t *) userp;
```

Comment:
Usually if a type have a _t in end, that means no struct is needed.

Either "struct curl_context" or curl_context_t.